### PR TITLE
minor spelling fixes and updating at Currency section

### PR DESCRIPTION
**Pull request by mauris**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -399,15 +399,16 @@ One of the two major passions (the other being eating) of Singaporeans.
...
-you can easily get cash out when you arive if nessary. While credit cards are
-widely accepted here, there are plenty of places which will only take cash.
-Certainly for public transport, cash is more heavily used. The best local food
-can be found in the older [hawker centres](#major-hawker-centres) and can't be
-bought with a credit card.
-
-Some retailers have a naughty habit of slapping on surcharges for credit
-card purchases, so, if you're shopping for say electronic goods, you might find
-that you get a small "discount" for paying by cash.
+you can easily get cash out at ATMs after you arrive if neesssary. While
```
Comment (winston) [inline README.md]: neesssary <<< :bow:

**Comment by cykhoo** [comment]
"Mostly harmless"

**Comment by mauris** [comment]
:sweat_smile:

File: README.md (hunk 1 above)
Comment (mauris) [inline README.md]: :scream:

**Pull request merged**